packages/core/botpress/src/botpress.js

```diff
@@ -86,6 +86,8 @@ const mkdirIfNeeded = (path, logger) => {
   }
 }
 
+const hasInspectMode = argv => argv.some(arg => /^(--inspect|-i)$/.test(arg))
+
```

Comment:
As we're already using commander doesn't it parse the args for us? So that we can check something like `options.inspect`?